src/Paket.Core/LockFile.fs

```diff
@@ -123,33 +124,34 @@ module LockFileSerializer =
                     if not (updateHasReported.Contains(GitHubLink)) then
                         yield "GITHUB"
                         updateHasReported.Remove (HttpLink "") |> ignore
-                        updateHasReported.Remove (GitLink "") |> ignore
+                        updateHasReported.Remove (GitLink (RemoteGitOrigin"")) |> ignore
```

Comment:
I'm not really sure how the serialization works here - are those changes relevant?